lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -13,10 +13,6 @@ def self.validate_ios_app(file_path)
               UI.user_error!("app verification failed: There must be only one .xctestrun files in the ZIP file.")
             end
 
-            conf = Plist.parse_xml(xctestrun_files.first.get_input_stream)
```

Comment:
Would removal of this line cause line 16 (_, scheme_conf = conf.first) to crash?

Reply:
Yeah, you're right, sorry i removed too much, i will put it back